gluetool/tests/test_eval_context.py

```diff
@@ -39,5 +39,5 @@ def test_module_via_glue(module, log):
     assert 'MODULE' in eval_context
     assert eval_context['MODULE'] is None
 
-    assert log.records[-3].message == 'Cannot infer calling module of eval_context'
-    assert log.records[-3].levelno == logging.WARNING
+    assert log.records[-1].message == 'Cannot infer calling module of eval_context'
```

Comment:
@happz maybe a comment would help why we need to do this

Reply:
Is it necessary? The patch is removing some verbose logs, which means that some log positions would be affected, as the logs, they expect to appear before them suddenly don't. This particular line does exactly that, two verbose entries no longer exist, therefore the warning is no longer 3rd entry counted from the end, but the first.

Reply:
thanks for the explanation, it would be nice to add it at least to commit message next time